webservices/resources/candidates.py

```diff
@@ -213,4 +237,58 @@ def get_candidate(self, args, page_num, per_page, candidate_id, committee_id):
         return count, candidates.order_by(CandidateDetail.expire_date.desc()).paginate(page_num, per_page, False).items
 
 
+class CandidateHistoryView(Resource):
+
+    parser = reqparse.RequestParser()
+    parser.add_argument('page', type=inputs.natural, default=1, help='For paginating through results, starting at page 1')
+    parser.add_argument('per_page', type=inputs.natural, default=20, help='The number of results returned per page. Defaults to 20.')
+    # We are standardizing on year
+    parser.add_argument('year', type=str, dest='year', help="See most recent record for a two-year cycle.")
```

Comment:
The information is not responsive but it would be helpful for filtering api calls. For example, if you wanted a list of incumbents that ran in 04 you could use the year and incumbent/challenger  = I to get those names.